src/content/docs/merchants/storefront-builder/content-commerce-blocks.mdx

```diff
@@ -0,0 +1,57 @@
+---
+title: Using Content and Commerce blocks
+description: Learn about the Content and Commerce blocks and how to use them to build your storefront.
+---
+
+**Content and Commerce blocks** are foundational, pre-built UI components that serve as the building blocks of your storefront. They are defined using tables in the content and decorated by your custom code during page rendering. Drop-ins are dynamic, commerce-specific front-end components that can be integrated within a Block.
+
+These blocks provide a modular approach to building storefronts, allowing you to quickly assemble pages using proven components while maintaining the flexibility to customize their appearance and behavior. As key components of the Storefront Builder ecosystem, they work seamlessly with Edge Delivery Services for optimal performance and can be easily managed through the Document Authoring tool.
+
+## What are Content and Commerce blocks?
```

Comment:
Consider adding a line noting that any block can leverage commerce functionality via Drop-ins. For example, the Header block uses the Cart and Auth Drop-ins, and this integration pattern applies broadly across blocks.